packages/modules/web-viewer-react/src/tests/services/Initializer.test.ts

```diff
@@ -126,4 +126,14 @@ describe("Initializer", () => {
     await WebInitializer.initialized;
     expect(IModelApp.startup).toHaveBeenCalled();
   });
+
+  it("IModel initialized before Webviewer initializer", async () => {
```

Comment:
maybe add a more descriptive description...think of it as a sentence that starts with "it" (i.e. it("should throw an error when..."))